charts/openshift-etcd-backup/values.yaml

```
@@ -0,0 +1,92 @@
+# Default values for openshift-etcd-backup.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+backup:
+  # Sub directory path
+  subdir: "/"
+  # Directory name of single backup
+  dirname: "+etcd-backup-%F-%H-%M-%S"
+  # expiretype could be days (keep backups newer than backup.keepdays,
+  # count (keep a number of backups with backup.keepcount),
+  # never (do not expire backups, keep all of them)
+  expiretype: "days"
+  # retention period
+  keepdays: "30"
+  # count retetion if expiretype set to count
+  keepcount: "10"
+  # backup schedule
+  schedule: "0 0 * * *"
+
+persistence:
+  # Set reclaim policy (Retain or Delete)
+  reclaimPolicy: Retain
+  # Define the storage size
+  capacity: 10Gi
+  nfs:
+    # Enable nfs backend storage
+    enabled: false
+    # NFS server name or IP
+    server: example.com
+    # NFS server path
+    path: "/etcd-backups"
+  hostPath:
+    # Enable hostPath
+    enabled: true
+    # hostPath existing path on host
+    path: "/opt/etcd-backups"
+  provisionned:
+    # Enable provisionned backend storage with default or overrided storageClass
+    enabled: false
+    storageClass: ""
+
+image:
+  # change repository value below with the result of the following command
+  # `oc adm release info --image-for=tools | cut -d: -f1`
+  repository: quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  # change tag value below with the result of the following command
+  # `oc adm release info --image-for=tools | cut -d: -f2`
+  tag: ""
+
+nameOverride: ""
+fullnameOverride: "etcd-backup"
```

should be empty per default
```suggestion
fullnameOverride: ""
```